web/src/js/components/Header/FilterInput.tsx

```diff
@@ -2,14 +2,16 @@ import React, { Component } from "react";
 import classnames from "classnames";
 import Filt from "../../filt/filt";
 import FilterDocs from "./FilterDocs";
+import WebsocketBackend from "../../backends/websocket";
+import BackendSingleton from "../../backends/BackendSingleton";
 
-type FilterInputProps = {
+interface FilterInputProps {
     type: string;
-    color: any;
+    color: string;
     placeholder: string;
     value: string;
-    onChange: (value) => void;
-};
+    onChange: (value: string) => void;
+}
```

Comment:
Thank you for improving type hints! 🍰 

Feel free to submit them as a separate micro PR so that we get them out of the critical path.